Offseason2022/src/main/java/frc/robot/subsystems/TowerConveyor.java

```diff
@@ -1,35 +1,87 @@
-
 // ROBOTBUILDER TYPE: Subsystem.
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatorCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.StatusFrame;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.wpilibj.DataLogManager;
 import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.frc2135.PhoenixUtil;
+import frc.robot.frc2135.RobotConfig;
+import frc.robot.Constants.TowerConveyor.Mode;
 
 /**
  *
  */
 public class TowerConveyor extends SubsystemBase
 {
-  private WPI_TalonFX  motorTC9;
-  private DigitalInput cargoDetect;
+  private boolean                         m_talonValidVC9;
+  private WPI_TalonFX                     m_motorVC9            = new WPI_TalonFX(9);
+  private double                          m_acquireSpeed;
+  private double                          m_acquireSpeedSlow;
+  private double                          m_expelSpeed;
+  private double                          m_expelSpeedFast;
+  private WPI_TalonFX                     m_cargoDetected;
+  private SupplyCurrentLimitConfiguration m_supplyCurrentLimits = new SupplyCurrentLimitConfiguration(true, 45.0, 45.0,
+      0.001);
+  private StatorCurrentLimitConfiguration m_statorCurrentLimits = new StatorCurrentLimitConfiguration(true, 80.0, 80.0,
+      0.001);
+  private final int                       kCANTimeout           = 30;
+  private int                             resetCountVC9;
 
   /**
    *
    */
   public TowerConveyor( )
   {
-    motorTC9 = new WPI_TalonFX(9);
+    // cargoDetect = new DigitalInput(0);
+    // addChild("CargoDetect", cargoDetect);
+
+    setName("Tower Conveyor");
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS

```

Comment:
Please remove any remaining RobotBuilder guards (lines that start with // BEGIN AUTOGENERATED... or // END AUTOGENERATED... These should not be copied from C++ code.